### PR TITLE
Updated approvers for winkernel proxy

### PR DESCRIPTION
**Pull request by madhanrm**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

**Comment by madhanrm** [comment]
FYI @dineshgovindasamy

**Comment by madhanrm** [comment]
@thockin  can you approve this, so that we will be able to approve the PRs in winkernel code?

File: pkg/proxy/winkernel/OWNERS [hunk 1]
```diff
@@ -1,3 +1,6 @@
+approvers:
+- dineshgovindasamy
+- madhanrm
```
Comment (feiskyer) [inline pkg/proxy/winkernel/OWNERS]: @madhanrm Also add me to the list? I'd like to improve this part in next release.

Comment (madhanrm) [reply pkg/proxy/winkernel/OWNERS]: done

**Comment by madhanrm** [comment]
Thanks @thockin

**Pull request merged**